docs/installation-guide/quickstart.rst

```diff
@@ -98,26 +103,36 @@ Prepare the bootstap node
 
 Install the bootstrap node
 --------------------------
-1. Run the script to install the bootstrap node.
+1. Run the bootstrap script to install binaries and services required on the
+   bootstrap node.
 
 .. code-block:: shell
 
    $ /srv/scality/metalk8s-2.0/bootstrap.sh
 
```

Comment:
we could indicate that progress can be followed in /var/log/metalk8s-bootstrap.log ... also reference to other salt logs could be interesting

Reply:
Good point